### ISSUE TITLE
Inteface const check

### ISSUE THREAD

**Issue by evans-xjw** [kalessil/phpinspectionsea#1714]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) or Php Inspections (EA Ultimate), which version |
| Language level | PHP 7.2 (check project settings or composer.json)                        |

Current behaviour
--
The subclass constants cover the constants of the interface parent class, but there is no prompt...

Expected behaviour
--
There should be a prompt

Environment details
--
Not needed

**Comment by ea-inspections-team** [comment]
Thank you for the proposal @evans-xjw! Would you please share a simplified code fragment, so we could understand why this is a problem? Thanks !

**Comment by evans-xjw** [comment]
```
interface ParentTestInterface
{
    public const TEST_INTERFACE_CONST = "I'm constant of interface";
}
```

```
class TestNormalClass implements ParentTestInterface
{
    public const TEST_INTERFACE_CONST = "I'm constant of subclass";
}
```

When echo `TestNormalClass ::TEST_INTERFACE_CONST`
Constant overlay error:  **Cannot inherit previously-inherited or override constant TEST_INTERFACE_CONST from interface...
But EA did not detect it**

**Comment by ea-inspections-team** [comment]
Thanks for sharing details @evans-xjw!
Looks like the latest PhpStrom already reports this.
